code/processes/gateway.q

```diff
@@ -100,10 +100,16 @@ queryqueue:([queryid:`u#`long$()] time:`timestamp$(); clienth:`g#`int$(); query:
 // client details
 clients:([]time:`timestamp$(); clienth:`g#`int$(); user:`symbol$(); ip:`int$(); host:`symbol$())
 
+//Function to generate random placeholder
+genrand:{system "S";-1?0Ng}
```

Comment:
Whats the purpose of `system"S"` here? Doesn't seem to do anything.

Reply:
OH, I was trying to reset the seed for the process but forgot to fix this function. Is this necessary or will it reset automatically dependent on time?

Reply:
The seed defaults to -314159 and I don't think it ever changes unless it's done explicitly.

Reply:
Ok, I'll have a look into it. Thanks Thomas!